lblod/berichtencentrum-sync-with-kalliope-service
# [Inzending] Adding extra filtering rules

# Description
DL-5305

This PR introduces new business rules : 

- EredienstBestuur (EB) is an organisation of CentraalBestuur (CB), when that's the case, the new rules should apply for the decision type of `Jaarrekening` and `Meerjarenplan`, meaning that every EB who has a CB that sends a submission with one of these two decision types will be excluded from Kalliope as it's supposed to not flow through Toezicht databank in future updates.

Same kind of logic will apply for the following :
- Centraal Bestuur (CB) who is a sender and has decision types `Budget(wijziging)`  and `Meerjarenplan(wijziging)`.
- Representatief Orgaan (RO) who is a sender and has decision type `Advies Budget(wijziging)`.
- Gemeente Overheid (GO) who is a sender and has decision types  `Besluit Budget(wijziging)` and `Besluit Meerjarenplan(wijziging)`.
- Provincie Overheid (PO) who is a sender and has decision types  `Besluit Budget(wijziging)` and `Besluit Meerjarenplan(wijziging)`.

Introduced change : 

- Updating the sparql query to apply this new rule in construct_unsent_inzendingen_query function.
- Creating a new function verify_inzendingen_flowthrough_rules to run checks and build a filter query where results are excluded from the main query in construct_unsent_inzendingen_query function.

# Type of change


# Related services

- Kalliope

# How to test 

1. Startup loket stack and create/send submissions with Eredienst Bestuur that has a CentraalBestuur and decision type of `Jaarrekening` or `Meerjarenplan` .
2. Run the query against the db to check if the submissions are excluded from the results
3. Same for all cases.

# What to check

- If business rules are correctly applied based on the rule sheet.

# Links to other PR's

- N/A

# Notes

- N/A
